### PR TITLE
DropTargets: QUICKFIX - fix Sluggish ball when rolling over dropped DT

### PR DESCRIPTION
**Pull request by Cupiii**
Does not fix triggering more than once when ball in hand and possibly even in normal play

### CODE REVIEW

File: VisualPinball.Unity/VisualPinball.Unity/Physics/Collision/StaticNarrowPhaseSystem.cs [hunk 1]
```diff
@@ -124,16 +127,44 @@ protected override void OnUpdate()
...
+								case ColliderType.Circle:
+								case ColliderType.LineZ:
+								case ColliderType.Plane:
+								case ColliderType.Point:
+								case ColliderType.Triangle:
+									// hit target
+									if (coll.Header.ItemType == ItemType.HitTarget) {
+										var normal = coll.Type == ColliderType.Triangle
+											? ((TriangleCollider*)collider)->Normal()
+											: collEvent.HitNormal;
```
Comment (freezy) [inline VisualPinball.Unity/VisualPinball.Unity/Physics/Collision/StaticNarrowPhaseSystem.cs:142]: `normal` isn't used, right? Can I remove it? (I'm cleaning up a few other things, too.)

File: VisualPinball.Unity/VisualPinball.Unity/Physics/Collision/StaticNarrowPhaseSystem.cs [hunk 2]
```diff
@@ -124,16 +127,44 @@ protected override void OnUpdate()
...
+										if (coll.Header.IsEnabled) { }
```
Comment (freezy) [inline VisualPinball.Unity/VisualPinball.Unity/Physics/Collision/StaticNarrowPhaseSystem.cs:143]: Also, this does nothing, good to remove?

**Review by freezy — APPROVED** [review]
Tested, and works great. Lemme know about the two comments, then I push my changes and merge.

Thanks! :)

File: VisualPinball.Unity/VisualPinball.Unity/Physics/Collision/StaticNarrowPhaseSystem.cs (hunk 1 above)
Comment (Cupiii) [reply VisualPinball.Unity/VisualPinball.Unity/Physics/Collision/StaticNarrowPhaseSystem.cs:142]: yeah, screw it.

File: VisualPinball.Unity/VisualPinball.Unity/Physics/Collision/StaticNarrowPhaseSystem.cs (hunk 2 above)
Comment (Cupiii) [reply VisualPinball.Unity/VisualPinball.Unity/Physics/Collision/StaticNarrowPhaseSystem.cs:143]: yup, you also can remove that.

**Comment by freezy** [comment]
All in. Cheers! :)

**Pull request merged**